k8s/helm/values.yaml

```diff
@@ -46,3 +46,5 @@ cdn:
 
 address_lookup_api:
   url: "https://whitelodge-ai-api.census-gcp.onsdigital.uk"
+
+confirmationEmailRequestLimit: 10
```

Comment:
This should be changed to `confirmationEmailLimit ` to be consistent with the environment variable change